packages/crlfuzz/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=crlfuzz
+pkgver=v1.4.1.r4.gc960f98
+pkgrel=1
+pkgdesc='A fast tool to scan CRLF vulnerability written in Go'
+url='https://github.com/dwisiswant0/crlfuzz'
+groups=('blackarch' 'blackarch-webapp')
+arch=('any')
+source=('git+https://github.com/dwisiswant0/crlfuzz.git')
+makedepends=('go')
+license=('MIT')
+sha512sums=('SKIP')
+
+pkgver(){
+  cd $pkgname
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
Always leave an empty line between sections like `cd` and other commands after it.
```suggestion
  cd $pkgname
  
  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```